<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="testcases.AddtocartRemove">
        <methods>
          <include name="Tc_SL_041"/>
          <include name="Login"/>
          <include name="quit"/>
        </methods>
      </class> <!-- testcases.AddtocartRemove -->
      <class name="testcases.ChkoutFail">
        <methods>
          <include name="quit"/>
          <include name="Tc_SL_050to053" invocation-numbers="0 1 2 3"/>
          <include name="Login"/>
        </methods>
      </class> <!-- testcases.ChkoutFail -->
      <class name="testcases.ChkoutwithoutProduct">
        <methods>
          <include name="quit"/>
          <include name="Tc_SL_045"/>
        </methods>
      </class> <!-- testcases.ChkoutwithoutProduct -->
      <class name="testcases.FilterandBack">
        <methods>
          <include name="Tc_SL_029"/>
          <include name="quit"/>
          <include name="Tc_SL_031"/>
          <include name="Tc_SL_030"/>
          <include name="Login"/>
        </methods>
      </class> <!-- testcases.FilterandBack -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
